### PR TITLE
docs: Update the Deploy Applications section of the OSM Manual Demo document

### PR DESCRIPTION
**Pull request by draychev**
This PR updates the **Installing OSM on Kubernetes** section of the "OSM Manual Demo" document.

In this PR:
  - add clarity and simplify verbiage
  - be explicit about the version of OSM (v0.8.2) this document is written for
  - swap `kubectl apply -f http://...` with `kubectl apply <<EOF heredoc EOF` -- this inlines the YAML in the document. I believe this allows us to look at what we are about to apply and understand in small chunks, without having to click through to another page (or copy paste the link, read etc).

---

This is one of a few steps to update the entire doc as part of #2845

Related PRs:
  - #3010
  - #3011 
  - #3012

---

Once the entire doc is rewritten - all changes will be applied to the `release-v0.8` branch.

### CODE REVIEW

**Review by SanyaKochhar — COMMENTED** [review]
Could you also update former line 208/now line 433:
```kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm/release-v0.8/docs/example/manifests/access/traffic-access-v1.yaml```

File: docs/content/docs/install/manual_demo/_index.md [hunk 1]
```diff
@@ -39,44 +39,269 @@ See the [observability documentation](../../patterns/observability/_index.md) fo
...
+|---------------|-----------|-----------|-----------|---------------|
+| bookbuyer     |     \     |     ❌     |     ✔    |       ❌       |
+| bookthief     |     ❌     |     \     |     ❌     |       ❌       |
+| bookstore     |     ❌     |     ❌     |     \     |       ✔      |
+| bookwarehouse |     ❌     |     ❌     |     ❌     |      \        |
+
+
+To show SMI Traffic Split, we will deploy an additional application:
+
+- `bookstore-v2` - a server, which is a newer version of the existing `bookstore` application
```
Comment (SanyaKochhar) [inline docs/content/docs/install/manual_demo/_index.md:66]: what does "newer" version imply? This line made me wonder about how it's different from v1

File: docs/content/docs/install/manual_demo/_index.md [hunk 2]
```diff
@@ -39,44 +39,269 @@ See the [observability documentation](../../patterns/observability/_index.md) fo
...
 ```bash
-for i in bookstore bookbuyer bookthief bookwarehouse; do kubectl create ns $i; done
+kubectl create namespace bookstore
+kubectl create namespace bookbuyer
+kubectl create namespace bookthief
+kubectl create namespace bookwarehouse
 ```
 
-### Onboard the Namespaces to the OSM Mesh and enable sidecar injection on the namespaces
+### Join the new namespaces to the service mesh
```
Comment (michelleN) [inline docs/content/docs/install/manual_demo/_index.md:80]: small nit/preference: "Add the new namespaces to the OSM control plane"

feel free to ignore if you don't agree.

File: docs/content/docs/install/manual_demo/_index.md [hunk 3]
```diff
@@ -39,44 +39,269 @@ See the [observability documentation](../../patterns/observability/_index.md) fo
...
+osm namespace add bookthief
+osm namespace add bookwarehouse
 ```
 
-### Create the Kubernetes resources for the bookstore demo applications
+Now each one of the four namespaces is labelled with `openservicemesh.io/monitored-by: osm` and also
+annotated with `openservicemesh.io/sidecar-injection: enabled`. The OSM Controller, noticing the label and annotation
+on these namespaces, will start injecting all **new** pods with Envoy sidecars.
+
+### Create the Pods and Services
```
Comment (michelleN) [inline docs/content/docs/install/manual_demo/_index.md:93]: what about the service accounts?

File: docs/content/docs/install/manual_demo/_index.md (hunk 1 above)
Comment (draychev) [reply docs/content/docs/install/manual_demo/_index.md:66]: Good question 🤔...
Let me take another stab at this.

Comment (draychev) [reply docs/content/docs/install/manual_demo/_index.md]: I reworded this to:

`bookstore-v2` - this is the same container as the first `bookstore` we deployed, but for this demo we will assume that it is a new version of the app we need to upgrade to.

File: docs/content/docs/install/manual_demo/_index.md (hunk 2 above)
Comment (draychev) [reply docs/content/docs/install/manual_demo/_index.md]: @michelleN I agree and I very much appreciate that.
I always think of `join namespace` and it sounds awkward (going to stop this going forward).
`Onboard` sounds too complicated.

`Add namespace` is simple and unambiguous!
Thank you!

File: docs/content/docs/install/manual_demo/_index.md (hunk 3 above)
Comment (draychev) [reply docs/content/docs/install/manual_demo/_index.md]: Added ServiceAccounts to the title!
Thank you!

File: docs/content/docs/install/manual_demo/_index.md (hunk 1 above)
Comment (SanyaKochhar) [reply docs/content/docs/install/manual_demo/_index.md]: yes, sounds good!

**Review by SanyaKochhar — APPROVED** [review]

**Comment by draychev** [comment]
@michelleN Could you PTAL?

**Review by snehachhabria — APPROVED** [review]

**Pull request merged**